src/DIS/ComputeDISOperators.f

```diff
@@ -141,6 +141,8 @@ subroutine ComputeDISOperators(Q)
          frac = 0.5d0
       elseif(TargetDIS(1:4).eq."iron")then
          frac = 0.47166350921037d0 !23.403d0 / 49.618d0
+      elseif(TargetDIS(1:4).eq."lead")then
+         frac = 0.396d0
```

Comment:
Looks good, but maybe I would just add a comment here explaining why/from where this number comes from.